.github/scripts/bump-minor-version/main.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+bump_minor_version() {
+    local version="$1"
+
+    if [[ $version =~ ^([0-9]+)\.([0-9]+)[0-9\.]*$ ]]; then
+        local major_version=${BASH_REMATCH[1]}
+        local minor_version=${BASH_REMATCH[2]}
```

Comment:
Nice! I didn't know this variable!

Reply:
Me neither until I wrote this 😅 it's quite useful!